_episodes/07-good-citizen.md

```diff
@@ -103,11 +103,19 @@ the full calculation.
 
 ## Have a backup plan
 
-Although the ARCHER2 /home file systems are backed up, the /work file systems are not,
-and /home is only backed up for disaster recovery purposes (*i.e.* for restoring the
-whole file system if lost rather than an individual file or directory you have deleted by mistake).
-Your data on ARCHER2 is primarily your responsibility and you should ensure you have secure copies of data
-that are critical to your work.
+The ARCHER2 /home file systems are backed up and are available at e.g. `/home1/.snapshot`.
+Within these `.snapshot` directories are hourly, daily and weekly backups, the layout within
```

Comment:
This section now correctly details the snapshots stored on the home file system, closing #23.